runtime/compiler/env/j9methodServer.cpp

```diff
@@ -2635,7 +2635,7 @@ TR_ResolvedRelocatableJ9JITServerMethod::validateMethodFieldAttributes(const TR_
    return equal;
    }
 
-TR_J9ServerMethod::TR_J9ServerMethod(TR_FrontEnd * fe, TR_Memory * trMemory, J9Class * aClazz, uintptr_t cpIndex)
+TR_J9ServerMethod::TR_J9ServerMethod(TR_FrontEnd * fe, TR_Memory * trMemory, J9Class * aClazz, int32_t cpIndex)
```

Comment:
As this constructor writes the value of `cpIndex` in a message below, I suspect that the code that reads the message in [`JITClientCompilationThread.cpp`](https://github.com/eclipse-openj9/openj9/blob/ee36c0cb5d07ca3c0bc9e9ba46b2d61c4647edf1/runtime/compiler/control/JITClientCompilationThread.cpp#L1453-L1459) will need to be updated to expect the same type of value.

@mpirvu, may I ask you to review this as well?

Reply:
Thanks, @hzongaro, I updated `JITClientCompilationThread.cpp` based on your comment.

Reply:
Because this also changes the wire format, I updated the minor version number in `CommunicationStream.hpp`.